src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java

```diff
@@ -131,33 +116,27 @@ public AceEditor getAceEditor() {
   }
 
   public void clearContent() {
-    driver.switchTo().frame(iframe);
-    contentInput.clear();
-    driver.switchTo().defaultContent();
+    editorFrame.frameScope(contentInput::clear);
   }
 
   /**
    * clears article content and adds new content to the article
    */
   public void addContent(String content) {
-    wait.forElementVisible(iframe);
-    driver.switchTo().frame(iframe);
-    contentInput.clear();
-    contentInput.sendKeys(content);
-    driver.switchTo().defaultContent();
+    editorFrame.frameScope(() -> contentInput.sendKeys(content));
     PageObjectLogging.log("addContent", "content " + content + " added to the article", true);
   }
 
+  public boolean checkPortableInfoboxVisible() {
+    return editorFrame.frameScope(portableInfoboxTransclusion::isDisplayed);
+  }
+
   private void verifyComponent(WebElement component) {
-    driver.switchTo().frame(iframe);
-    wait.forElementVisible(component);
-    driver.switchTo().defaultContent();
+    editorFrame.frameScope(() -> wait.forElementVisible(component));
   }
 
   private void verifyComponent(By componentBy) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "verifyComponent" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
@TK-999 ☝️

Reply:
sonar false positive